### PR TITLE
Create Parent - Child relationship property class

### LINKED ISSUE #359: Create Parent Child Relationships property class
### Problem Description

Once #357 is done, we can migrate the code for computing the parent child relationship scores to a property class.

#### Attributes

- `metrics`: [CardinalityShapeSimilarity]
- `_properties`: None

#### Abstract methods

- `get_score(real_data, synthetic_data, metadata)` - Average the scores calculated when running the CardinalityShapeSimilarity metric.
- `get_visualization(table_name)` - Just a wrapper around the single table `get_visualization` method.

### Additional context
- Unlike the other properties, this one has no single table equivalent. The metric itself is designed to handle multi table data, so this class will not have to use any single table properties.

### PR DESCRIPTION
**Pull request by pvk-developer**
Resolves #359

### CODE REVIEW

**Review by frances-h — APPROVED** [review]

File: sdmetrics/reports/multi_table/_properties/relationship.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+from sdmetrics.multi_table.statistical import CardinalityShapeSimilarity
+from sdmetrics.reports.multi_table._properties.base import BaseMultiTableProperty
+from sdmetrics.reports.multi_table.plot_utils import get_table_relationships_plot
+
+
+class CardinalityShapeSimilarityProperty(BaseMultiTableProperty):
```
Comment (amontanez24) [inline sdmetrics/reports/multi_table/_properties/relationship.py:6]: We can just call it `Cardinality`. Can we also rename the file to match?

File: tests/unit/reports/multi_table/_properties/test_relationship.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            ('users', 'sessions'): 1.,
+            ('users', 'transactions'): 0.5
+        }
+
+        # Run
+        result = CardinalityShapeSimilarityProperty.get_visualization(instance, 'sessions')
+
+        # Assert
+        assert result == mock_get_table_relationships_plot.return_value
+        expected_score_breakdwons = {
```
Comment (amontanez24) [inline tests/unit/reports/multi_table/_properties/test_relationship.py:84]: typo: breakdwons

File: sdmetrics/reports/multi_table/_properties/relationship.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        for relation in metadata.get('relationships', []):
+            relationships_metadata = {'relationships': [relation]}
+            try:
+                self._metric_results = CardinalityShapeSimilarity.compute_breakdown(
+                    real_data,
+                    synthetic_data,
+                    relationships_metadata
+                )
+            except Exception:
+                pass
```
Comment (amontanez24) [inline sdmetrics/reports/multi_table/_properties/relationship.py:42]: I think in the other metrics we are appending the error message to a new column with the header `Error`. We should do the same. That is the old behavior too I believe

**Review by amontanez24 — CHANGES_REQUESTED** [review]

**Review by amontanez24 — APPROVED** [review]
LGTM! Thanks for the changes :shipit:

**Pull request merged**